src/allmydata/test/test_testing.py

```diff
@@ -0,0 +1,38 @@
+from twisted.trial.unittest import (
```

Comment:
need a copyright statement and module docstring

Reply:
I could only find two other modules with a copyright notice .. most don't have one. I did copy it, but .. *does* every file need one?

Reply:
I don't believe it is a legal requirement but I think it is strongly encouraged because individual files are an easily separated unit of "work of authorship".

I forgot Tahoe-LAFS source code generally doesn't do this and asked for it by reflex.  The horse may be out of the barn on this one already.  I don't believe we are on firm ground going back and adding such a statement to all existing files.  Instead, we need to establish authorship and attribute copyright appropriately (or obtain copyright assignment).

So ... this doesn't hurt for new files but it might not really buy the project much, either.  It's easy though so maybe it's still worth doing.